auth-web/src/util/constants.ts

```diff
@@ -247,6 +247,87 @@ export enum NrEntityType {
     INFO = 'INFO', // special value for sub-menu
 }
 
+export enum NrRequestTypes {
```

Comment:
These aren't actually NR Request Types -- they're the strings for the types. (Just trying to avoid future confusion.)

Reply:
Would a name like NrRequestTypeStrings make more sense then, or would a comment suffice?

Reply:
Well, it's not a list of codes and it's not a list of types, so yes, even though it's wordy.

[Naming things is one of the hard things in computer science.](https://blog.frankel.ch/hard-things-computer-science/)

Reply:
LOL -^ I was just talking about this the other day with Cam